src/app/compressed-air-assessment/inventory/performance-points/performance-points.component.html

```diff
@@ -2,8 +2,38 @@
     <span class="fa fa-chevron-right" [ngClass]="{'fa-chevron-right': contentCollapsed, 'fa-chevron-down': !contentCollapsed}"></span>
     Performance Points
 </label>
-<app-full-load *ngIf="!contentCollapsed"></app-full-load>
-<app-max-full-flow *ngIf="showMaxFullFlow && !contentCollapsed"></app-max-full-flow>
-<app-unload-point *ngIf="showUnload && !contentCollapsed"></app-unload-point>
-<app-no-load *ngIf="showNoLoad && !contentCollapsed"></app-no-load>
-<app-blowoff *ngIf="showBlowoff && !contentCollapsed"></app-blowoff>
\ No newline at end of file
+
+<div *ngIf="!contentCollapsed" class="p-2">
+    <table class="table">
+        <thead>
+            <tr class="small bold">
+                <th class="p-1"></th>
+                <th class="text-center p-1">
+                    Discharge Pressure
+                </th>
+                <th class="text-center p-1">
+                    Airflow
+                </th>
+                <th class="text-center p-1">
+                    Power
+                </th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr [app-full-load]></tr>
+            <tr [app-max-full-flow] *ngIf="showMaxFullFlow"></tr>
+            <tr [app-unload-point] *ngIf="showUnload"></tr>
+            <tr [app-no-load] *ngIf="showNoLoad"></tr>
+            <tr [app-blowoff] *ngIf="showBlowoff"></tr>
+        </tbody>
+    </table>
+</div>
+
+<!-- @input isSmallScreen and use original html form template-->
+<ng-container *ngIf="isSmallScreen">
+    <div [isSmallScreen]="isSmallScreen" [app-full-load]></div>
+    <div [isSmallScreen]="isSmallScreen" [app-max-full-flow] *ngIf="showMaxFullFlow"></div>
+    <div [isSmallScreen]="isSmallScreen" [app-unload-point] *ngIf="showUnload"></div>
+    <div [isSmallScreen]="isSmallScreen" [app-no-load] *ngIf="showNoLoad"></div>
+    <div [isSmallScreen]="isSmallScreen" [app-blowoff] *ngIf="showBlowoff"></div>
+</ng-container>
```

Comment:
I don't see a isSmallScreen variable anywhere in .ts. Maybe I'm missing something..